### ISSUE TITLE
Feature Request: Allow adding and signing of additional pbo files

### ISSUE THREAD

**Issue by bux** [BrettMayson/HEMTT#64]
**HEMTT Version:** `0.5.1` (stable)

As a user I would like to add 3rd party pbo files to e.g. a separate folder and while building have them copied to the addons folder and have them signed with the same private key to ensure that all community members use the proper version of a 3rd party mod.

**Comment by jonpas** [comment]
Similar or almost the same as #41, @bux move discussion there?

**Comment by bux** [comment]
If the result of it will be applicable for that request, go for it :)

**Comment by jonpas** [comment]
I don't think we'll provide full setup for exactly what you want, but just splitting sign step out so you can write your own scripts inside HEMTT configuration for it. As there is no way we can satisfy everyone's setup by doing it in any more elaborate way.

**Comment by synixebrett** [comment]
Adding wontfix but I am going to keep this open until #78 is closed.

Once #78 is done you will be able to do this using a HEMTT script to call `armake sign`

**Comment by bux** [comment]
I guess this can be closed now?
